clinica/utils/input_files.py

```diff
@@ -0,0 +1,128 @@
+# coding: utf8
+
+"""
+Describe files to grab, to use with inputs.clinica_file_reader() and inputs.clinica_group_reader()
+
+"""
+
+""" T1w """
+
+# BIDS
+
+T1W_NII = {'pattern': 'sub-*_ses-*_t1w.nii*',
+           'description': 'T1w MRI'}
+
+# FreeSurfer
+
+T1_FS_WM = {'pattern': 'sub-*_ses-*/mri/wm.seg.mgz',
+            'description': 'segmentation of white matter (mri/wm.seg.mgz)',
+            'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_BRAIN = {'pattern': 'sub-*_ses-*/mri/brain.mgz',
+               'description': ' extracted brain from T1w MRI',
+               'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_ORIG_NU = {'pattern': 'mri/orig_nu.mgz',
+                 'description': 'intensity normalized volume generated after correction for'
+                                ' non-uniformity in FreeSurfer (file .../mri/orig_nu.mgz) ',
+                 'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_WS_R = {'pattern': 'sub-*_ses-*/surf/rh.white',
+              'description': 'right hemisphere of outter cortical surface.',
```

Comment:
Wrong ;). Proposition:
```suggestion
              'description': 'right white matter/gray matter border surface.',
```

Reply:
Done